persistence-modules/pom.xml

```diff
@@ -14,12 +14,10 @@
     </parent>
 
     <modules>
-        <module>activejdbc</module>
         <module>apache-bookkeeper</module>
         <module>apache-cayenne</module>
         <module>apache-derby</module>
         <module>blaze-persistence</module>
-        <module>core-java-persistence</module>
```

Comment:
let's not do this, let's do that reverse.
Move persistence-modules into jdk9 in the main pom.xml
and in this pom.xml those that are being migrated enabled it and keep others as disabled. Gradually we will enable all of them as we make progress on this Jira

Reply:
I will do the changes in the next commit